Kudu.Services/Deployment/DeploymentController.cs

```diff
@@ -111,8 +111,9 @@ public void Delete(string id)
                             {
                                 CreateDeployment(deployResult, result.Value<string>("details"));
 
-                                deployResult.Url = Request.RequestUri;
-                                deployResult.LogUrl = UriHelper.MakeRelative(Request.RequestUri, "log");
+                                Uri baseUri = UriHelper.MakeRelative(UriHelper.GetBaseUri(Request), Request.RequestUri.AbsolutePath);
```

Comment:
I find that having specific examples as comment showing what string handling is doing really helps read through the code.